### PR TITLE
Minor tidy website update

### PR DESCRIPTION
**Pull request by NoubissiViany**
## Description

- Minor tidy website update
- Updated all header pages
- Updated all footer pages 

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [x] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

## Changes

Please which part of the project your changes affect ?

- [x] Frontend (Templates, CSS, JS)
- [ ] Backend (Dancer2, api-docs, Dockerfile)

### CODE REVIEW

File: prototype/Frontend/about.html [hunk 1]
```diff
@@ -100,14 +100,14 @@
         >
           <a href="index.html" class="logo me-3">
             <img
-              src="/public/images/logo/logo.png"
+              src="../../public/images/logo/logo.png"
```
Comment (manwar) [inline prototype/Frontend/about.html]: No relative path please.

File: prototype/Frontend/about.html [hunk 2]
```diff
@@ -100,14 +100,14 @@
...
               alt="logo"
               class="img-fluid"
             />
           </a>
           <a href="index.html" class="logo me-auto">
             <img
-              src="/public/images/logo/typography.png"
+              src="../../public/images/logo/typography.png"
```
Comment (manwar) [inline prototype/Frontend/about.html]: No relative path please.

File: prototype/Frontend/about.html [hunk 3]
```diff
@@ -124,7 +124,7 @@
             <!-- About Dropdown -->
             <li class="dropdown">
               <a href="#">
-                <span>About</span> <i class="bi bi-chevron-down"></i>
+                <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/about.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/blogs.html [hunk 4]
```diff
@@ -106,14 +106,14 @@
         >
           <a href="index.html" class="logo me-3">
             <img
-              src="/public/images/logo/logo.png"
+              src="../../public/images/logo/logo.png"
```
Comment (manwar) [inline prototype/Frontend/blogs.html]: No relative path please.

File: prototype/Frontend/blogs.html [hunk 5]
```diff
@@ -106,14 +106,14 @@
...
               alt="logo"
               class="img-fluid"
             />
           </a>
           <a href="index.html" class="logo me-auto">
             <img
-              src="/public/images/logo/typography.png"
+              src="../../public/images/logo/typography.png"
```
Comment (manwar) [inline prototype/Frontend/blogs.html]: No relative path please.

File: prototype/Frontend/blogs.html [hunk 6]
```diff
@@ -130,7 +130,7 @@
             <!-- About Dropdown -->
             <li class="dropdown">
               <a href="#">
-                <span>About</span> <i class="bi bi-chevron-down"></i>
+                <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/blogs.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/champions.html [hunk 7]
```diff
@@ -103,14 +103,14 @@
         >
           <a href="index.html" class="logo me-3">
             <img
-              src="/public/images/logo/logo.png"
+              src="../../public/images/logo/logo.png"
```
Comment (manwar) [inline prototype/Frontend/champions.html]: No relative path please.

File: prototype/Frontend/champions.html [hunk 8]
```diff
@@ -103,14 +103,14 @@
...
               alt="logo"
               class="img-fluid"
             />
           </a>
           <a href="index.html" class="logo me-auto">
             <img
-              src="/public/images/logo/typography.png"
+              src="../../public/images/logo/typography.png"
```
Comment (manwar) [inline prototype/Frontend/champions.html]: No relative path please.

File: prototype/Frontend/champions.html [hunk 9]
```diff
@@ -127,7 +127,7 @@
             <!-- About Dropdown -->
             <li class="dropdown">
               <a href="#">
-                <span>About</span> <i class="bi bi-chevron-down"></i>
+                <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/champions.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/contact.html [hunk 10]
```diff
@@ -121,7 +121,7 @@
             <!-- About Dropdown -->
             <li class="dropdown">
               <a href="#">
-                <span>About</span> <i class="bi bi-chevron-down"></i>
+                <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/contact.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/create-new-blog-post.html [hunk 11]
```diff
@@ -125,7 +125,7 @@
             <!-- About Dropdown -->
             <li class="dropdown">
               <a href="#">
-                <span>About</span> <i class="bi bi-chevron-down"></i>
+                <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/create-new-blog-post.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/faq.html [hunk 12]
```diff
@@ -125,7 +125,7 @@
             <!-- About Dropdown -->
             <li class="dropdown">
               <a href="#">
-                <span>About</span> <i class="bi bi-chevron-down"></i>
+                <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/faq.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/guest-leader.html [hunk 13]
```diff
@@ -120,7 +120,7 @@
             <!-- About Dropdown -->
             <li class="dropdown">
               <a href="#">
-                <span>About</span> <i class="bi bi-chevron-down"></i>
+                <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/guest-leader.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/guest.html [hunk 14]
```diff
@@ -119,7 +119,7 @@
             <!-- About Dropdown -->
             <li class="dropdown">
               <a href="#">
-                <span>About</span> <i class="bi bi-chevron-down"></i>
+                <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/guest.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/index.html [hunk 15]
```diff
@@ -163,7 +163,7 @@
             <!-- About Dropdown -->
             <li class="dropdown">
               <a href="#">
-                <span>About</span> <i class="bi bi-chevron-down"></i>
+                <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/index.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/list-outreachy-applicants.html [hunk 16]
```diff
@@ -173,7 +173,7 @@
             <!-- About Dropdown -->
             <li class="dropdown">
               <a href="#">
-                <span>About</span> <i class="bi bi-chevron-down"></i>
+                <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/list-outreachy-applicants.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/logout.html [hunk 17]
```diff
@@ -121,7 +121,7 @@
             <!-- About Dropdown -->
             <li class="dropdown">
               <a href="#">
-                <span>About</span> <i class="bi bi-chevron-down"></i>
+                <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/logout.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/new-perl.html [hunk 18]
```diff
@@ -123,7 +123,7 @@
             <!-- About Dropdown -->
             <li class="dropdown">
               <a href="#">
-                <span>About</span> <i class="bi bi-chevron-down"></i>
+                <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/new-perl.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/new-raku.html [hunk 19]
```diff
@@ -126,7 +126,7 @@
             <!-- About Dropdown -->
             <li class="dropdown">
               <a href="#">
-                <span>About</span> <i class="bi bi-chevron-down"></i>
+                <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/new-raku.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/perl-review.html [hunk 20]
```diff
@@ -129,7 +129,7 @@
             <!-- About Dropdown -->
             <li class="dropdown">
               <a href="#">
-                <span>About</span> <i class="bi bi-chevron-down"></i>
+                <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/perl-review.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/raku-review.html [hunk 21]
```diff
@@ -130,7 +130,7 @@
             <!-- About Dropdown -->
             <li class="dropdown">
               <a href="#">
-                <span>About</span> <i class="bi bi-chevron-down"></i>
+                <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/raku-review.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/read-post.html [hunk 22]
```diff
@@ -125,7 +125,7 @@
             <!-- About Dropdown -->
             <li class="dropdown">
               <a href="#">
-                <span>About</span> <i class="bi bi-chevron-down"></i>
+                <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/read-post.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/rss_feed.html [hunk 23]
```diff
@@ -126,7 +126,7 @@
             <!-- About Dropdown -->
             <li class="dropdown">
               <a href="#">
-                <span>About</span> <i class="bi bi-chevron-down"></i>
+                <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/rss_feed.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/search-weekly-challenges.html [hunk 24]
```diff
@@ -119,7 +119,7 @@
             <!-- About Dropdown -->
             <li class="dropdown">
               <a href="#">
-                <span>About</span> <i class="bi bi-chevron-down"></i>
+                <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/search-weekly-challenges.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/search_contributions.html [hunk 25]
```diff
@@ -180,7 +180,7 @@
             <!-- About Dropdown -->
             <li class="dropdown">
               <a href="#">
-                <span>About</span> <i class="bi bi-chevron-down"></i>
+                <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/search_contributions.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/sign-in.html [hunk 26]
```diff
@@ -121,7 +121,7 @@
             <!-- About Dropdown -->
             <li class="dropdown">
               <a href="#">
-                <span>About</span> <i class="bi bi-chevron-down"></i>
+                <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/sign-in.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/submit-contribution.html [hunk 27]
```diff
@@ -124,7 +124,7 @@
             <!-- About Dropdown -->
             <li class="dropdown">
               <a href="#">
-                <span>About</span> <i class="bi bi-chevron-down"></i>
+                <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/submit-contribution.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/team-leader.html [hunk 28]
```diff
@@ -120,7 +120,7 @@
             <!-- About Dropdown -->
             <li class="dropdown">
               <a href="#">
-                <span>About</span> <i class="bi bi-chevron-down"></i>
+                <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/team-leader.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/team.html [hunk 29]
```diff
@@ -120,7 +120,7 @@
             <!-- About Dropdown -->
             <li class="dropdown">
               <a href="#">
-                <span>About</span> <i class="bi bi-chevron-down"></i>
+                <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/team.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/user_dashboard.html [hunk 30]
```diff
@@ -128,7 +128,7 @@
               <!-- About Dropdown -->
               <li class="dropdown">
                 <a href="#">
-                  <span>About</span> <i class="bi bi-chevron-down"></i>
+                  <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/user_dashboard.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/user_replies.html [hunk 31]
```diff
@@ -133,7 +133,7 @@
               <!-- About Dropdown -->
               <li class="dropdown">
                 <a href="#">
-                  <span>About</span> <i class="bi bi-chevron-down"></i>
+                  <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/user_replies.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/user_view_comment.html [hunk 32]
```diff
@@ -133,7 +133,7 @@
               <!-- About Dropdown -->
               <li class="dropdown">
                 <a href="#">
-                  <span>About</span> <i class="bi bi-chevron-down"></i>
+                  <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/user_view_comment.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/view-weekly-challenges.html [hunk 33]
```diff
@@ -118,7 +118,7 @@
             <!-- About Dropdown -->
             <li class="dropdown">
               <a href="#">
-                <span>About</span> <i class="bi bi-chevron-down"></i>
+                <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/view-weekly-challenges.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/view_contributions.html [hunk 34]
```diff
@@ -224,7 +224,7 @@
             <!-- About Dropdown -->
             <li class="dropdown">
               <a href="#">
-                <span>About</span> <i class="bi bi-chevron-down"></i>
+                <span>Learn More</span> <i class="bi bi-chevron-down"></i>
```
Comment (manwar) [inline prototype/Frontend/view_contributions.html]: Nice choice but please make it like "Learn more ..." instead.

File: prototype/Frontend/contact.html [hunk 35]
```diff
@@ -97,14 +97,14 @@
         >
           <a href="index.html" class="logo me-3">
             <img
-              src="/public/images/logo/logo.png"
+              src="../../public/images/logo/logo.png"
```
Comment (manwar) [inline prototype/Frontend/contact.html]: No relative path please.

File: prototype/Frontend/contact.html [hunk 36]
```diff
@@ -97,14 +97,14 @@
...
               alt="logo"
               class="img-fluid"
             />
           </a>
           <a href="index.html" class="logo me-auto">
             <img
-              src="/public/images/logo/typography.png"
+              src="../../public/images/logo/typography.png"
```
Comment (manwar) [inline prototype/Frontend/contact.html]: No relative path please.

File: prototype/Frontend/create-new-blog-post.html [hunk 37]
```diff
@@ -101,14 +101,14 @@
         >
           <a href="index.html" class="logo me-3">
             <img
-              src="/public/images/logo/logo.png"
+              src="../../public/images/logo/logo.png"
```
Comment (manwar) [inline prototype/Frontend/create-new-blog-post.html]: No relative path please.

File: prototype/Frontend/create-new-blog-post.html [hunk 38]
```diff
@@ -101,14 +101,14 @@
...
               alt="logo"
               class="img-fluid"
             />
           </a>
           <a href="index.html" class="logo me-auto">
             <img
-              src="/public/images/logo/typography.png"
+              src="../../public/images/logo/typography.png"
```
Comment (manwar) [inline prototype/Frontend/create-new-blog-post.html]: No relative path please.

File: prototype/Frontend/faq.html [hunk 39]
```diff
@@ -101,14 +101,14 @@
         >
           <a href="index.html" class="logo me-3">
             <img
-              src="/public/images/logo/logo.png"
+              src="../../public/images/logo/logo.png"
```
Comment (manwar) [inline prototype/Frontend/faq.html]: No relative path please.

File: prototype/Frontend/faq.html [hunk 40]
```diff
@@ -101,14 +101,14 @@
...
               alt="logo"
               class="img-fluid"
             />
           </a>
           <a href="index.html" class="logo me-auto">
             <img
-              src="/public/images/logo/typography.png"
+              src="../../public/images/logo/typography.png"
```
Comment (manwar) [inline prototype/Frontend/faq.html]: No relative path please.

File: prototype/Frontend/guest-leader.html [hunk 41]
```diff
@@ -96,14 +96,14 @@
         >
           <a href="index.html" class="logo me-3">
             <img
-              src="/public/images/logo/logo.png"
+              src="../../public/images/logo/logo.png"
```
Comment (manwar) [inline prototype/Frontend/guest-leader.html]: No relative path please.

File: prototype/Frontend/guest-leader.html [hunk 42]
```diff
@@ -96,14 +96,14 @@
...
               alt="logo"
               class="img-fluid"
             />
           </a>
           <a href="index.html" class="logo me-auto">
             <img
-              src="/public/images/logo/typography.png"
+              src="../../public/images/logo/typography.png"
```
Comment (manwar) [inline prototype/Frontend/guest-leader.html]: No relative path please.

File: prototype/Frontend/guest.html [hunk 43]
```diff
@@ -95,14 +95,14 @@
         >
           <a href="index.html" class="logo me-3">
             <img
-              src="/public/images/logo/logo.png"
+              src="../../public/images/logo/logo.png"
```
Comment (manwar) [inline prototype/Frontend/guest.html]: No relative path please.

File: prototype/Frontend/guest.html [hunk 44]
```diff
@@ -95,14 +95,14 @@
...
               alt="logo"
               class="img-fluid"
             />
           </a>
           <a href="index.html" class="logo me-auto">
             <img
-              src="/public/images/logo/typography.png"
+              src="../../public/images/logo/typography.png"
```
Comment (manwar) [inline prototype/Frontend/guest.html]: No relative path please.

File: prototype/Frontend/list-outreachy-applicants.html [hunk 45]
```diff
@@ -149,14 +149,14 @@
         >
           <a href="index.html" class="logo me-3">
             <img
-              src="/public/images/logo/logo.png"
+              src="../../public/images/logo/logo.png"
```
Comment (manwar) [inline prototype/Frontend/list-outreachy-applicants.html]: No relative path please.

File: prototype/Frontend/list-outreachy-applicants.html [hunk 46]
```diff
@@ -149,14 +149,14 @@
...
               alt="logo"
               class="img-fluid"
             />
           </a>
           <a href="index.html" class="logo me-auto">
             <img
-              src="/public/images/logo/typography.png"
+              src="../../public/images/logo/typography.png"
```
Comment (manwar) [inline prototype/Frontend/list-outreachy-applicants.html]: No relative path please.

File: prototype/Frontend/logout.html [hunk 47]
```diff
@@ -97,14 +97,14 @@
         >
           <a href="index.html" class="logo me-3">
             <img
-              src="/public/images/logo/logo.png"
+              src="../../public/images/logo/logo.png"
```
Comment (manwar) [inline prototype/Frontend/logout.html]: No relative path please.

File: prototype/Frontend/logout.html [hunk 48]
```diff
@@ -97,14 +97,14 @@
...
               alt="logo"
               class="img-fluid"
             />
           </a>
           <a href="index.html" class="logo me-auto">
             <img
-              src="/public/images/logo/typography.png"
+              src="../../public/images/logo/typography.png"
```
Comment (manwar) [inline prototype/Frontend/logout.html]: No relative path please.

File: prototype/Frontend/new-perl.html [hunk 49]
```diff
@@ -99,14 +99,14 @@
         >
           <a href="index.html" class="logo me-3">
             <img
-              src="/public/images/logo/logo.png"
+              src="../../public/images/logo/logo.png"
```
Comment (manwar) [inline prototype/Frontend/new-perl.html]: No relative path please.

File: prototype/Frontend/new-perl.html [hunk 50]
```diff
@@ -99,14 +99,14 @@
...
               alt="logo"
               class="img-fluid"
             />
           </a>
           <a href="index.html" class="logo me-auto">
             <img
-              src="/public/images/logo/typography.png"
+              src="../../public/images/logo/typography.png"
```
Comment (manwar) [inline prototype/Frontend/new-perl.html]: No relative path please.

**Review by manwar — COMMENTED** [review]
Good job.

**Comment by manwar** [comment]
Thanks for your contributions. @NoubissiViany

**Comment by manwar** [comment]
@NoubissiViany just noticed, logo is duplicated.
image

**Comment by NoubissiViany** [comment]
@manwar i'm fixing that

**Pull request merged**